### DISCUSSION TITLE
Svelte example

### DISCUSSION THREAD

**Discussion by liamwh** [loco-rs/loco#644] [General]
Disclosure: I'm relatively new to web-dev.

I've spent quite some time trying to get Svelte working with Loco.rs but I'm not having much success. I've tried using both rsbuild and vite but haven't been able to get things working properly, most likely due to my lack of experience with web dev. Would really appreciate an example app that uses Svelte instead of React to help me get going!

Thanks,

Liam

**Comment by kaplanelad** [comment]
Can you provide a repo that i can take a look?
